machine_management/control_plane_machine_management/cpmso-using.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="cpmso-using"]
+= Managing control plane machines with the Control Plane Machine Set Operator
```

Comment:
This section is commented out in the build and must be reviewed in a later PR that finishes this section.